train.py

```diff
@@ -224,7 +226,9 @@ def main():
     reader.start_threads(sess)
 
     try:
+        last_save_step = None
```

Comment:
Maybe `last_save_step = saved_global_step` is more stable, considering training on restored checkpoint.